lambda_functions/execute_ssh_command_js/README.md

```diff
@@ -1,13 +1,72 @@
-
-
 ## Setup
 
 Since this plugin uses the Serverless plugin `serverless-secrets-plugin` you need to setup the `node_modules` by running:
 
-    npm install
+    1) Follow nvm installation guide as mentioned in https://github.com/creationix/nvm. 
+    2) cd to the folder where nvm.sh is installed and run the following command to source to ~/.bashrc, or ~/.profile, or ~/.zshrc file:
+          `. nvm.sh`
+    3) Run the following shell commands (to check if installation is completed):
+        a) `nvm --version`
+        b) `nvm install node`
+        c) `nvm use node`
+        d) `node --version`
+        e) `nvm install npm`
+        f) `npm --version`
+        g) `npm install -g serverless`
+        h) `serverless --version`
+        i) `serverless config credentials --provider aws --key <AWS key> --secret <AWS Secret key>`
+
+## Installation before deployment
+
+    1) Run the following shell commands (to install dependencies):
+        a) `npm install serverless-pseudo-parameters --save-dev`
+        b) `npm install simple-ssh --save-dev`
```

Comment:
The way `node/npm` handles dependencies, means that we should be able to run `npm install` without the extra parameters. It looks inside `package.json` for which packages to install.

`--save` or `--save-dev` is an instruction to add the package into the `package.json` file. Similar to `requirements.txt` in python land.